docs/modeling/processes/bpmn/user.md

```diff
@@ -3,26 +3,133 @@ Title: User tasks
 
-[Process variables](../variables.md) can be mapped from the originating process to [form fields](../../forms/fields.md) and [form variables](../../forms/README.md#form-variables) in a user task as inputs. Similarly, form field data from the completed user task can be transferred back to process variables in the process as outputs. The mapping between process variables and form variables is stored in the `<process-name>-extensions.json` file for the process definition under the `mappings` section. 
+User tasks are displayed as a single thin, rounded rectangle with a user icon inside.
 
-The possible assignments for a user task are:
+## XML
+The XML for a user task is similar to the following:
 
-* Assignee: a specific user is defined.
-* Candidate user: a list of users that may claim the task. 
-* Candidate group: a group(s) of users that can claim the task. 
+```xml
+<bpmn2:userTask id="UserTask_0gpdh83" name="Order" activiti:formKey="form-38098a3e-bff1-46cb-ba0f-0c94fdb287ed" activiti:assignee="${userDetails.username}" activiti:dueDate="2020-01-01T01:00:00" activiti:priority="2">
+	<bpmn2:documentation>A form to choose the flavor of ice cream.</bpmn2:documentation>
+	<bpmn2:incoming>SequenceFlow_02eaofe</bpmn2:incoming>
+	<bpmn2:outgoing>SequenceFlow_14ma5mo</bpmn2:outgoing>
+</bpmn2:userTask>
+``` 
+
+## Properties 
+
+### Basic properties
+The basic properties for a user task are: 
+
+| Property | Description | Example | Required | 
+| -------- | ----------- | ------- | -------- | 
+| `id` | The unique identifier for the user task. This is system generated and cannot be altered | UserTask_0gpdh83 | Yes |
+| `name` | The name of the user task. This will display on the user task in the process diagram | Flavor order | No |
+| `documentation` | A free text description of what the user task does | A form to choose the flavor of ice cream.  | No |
+
+The ID and name of a user task are set as XML attributes of the `userTask`. Documentation is a sub-element of `userTask`, for example: 
+
+```xml
+<bpmn2:userTask id="UserTask_0gpdh83" name="Order">
+	<bpmn2:documentation>A form to choose the flavor of ice cream.</bpmn2:documentation>
+</bpmn2:userTask>
+```
+
+### Assignment
+The users or groups that are able to complete a task. A single user can be assigned or candidates can be set. Candidates are a list of users or groups that may [claim a task](../../../workspace/tasks.md#claiming-a-task) at runtime. A single user or candidates must be set for a user task.  
+
+A single assignee is set in the XML attribute `activiti:assignee` of the `userTask`, for example: 
+
+```xml
+<bpmn2:userTask activiti:assignee="hruser"> 
+```
+
+Candidates are set in the XML attribute `activiti:candidateGroups` of the `userTask`, for example:
+
+```xml
+<bpmn2:userTask activiti:candidateGroups="hrgroup,${groupDetails.groupNames}">
```

Comment:
it would be better to not mix the static with expression.

Reply:
Updated the example to just contain an expression